### PR TITLE
CI environment refinements

### PR DESCRIPTION
**Pull request by larroy**
## Description ##
Don't install nvidia driver
Don't install gtest 
set -x and -e  so scripts fail when command fail.

### Essentials ###
- [ ] Passed code style checking (`make lint`)
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [ ] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- [ ] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

**Comment by larroy** [comment]
@marcoabreu

File: tests/ci_build/install/ubuntu_install_nvidia.sh [hunk 1]
```diff
@@ -19,13 +19,19 @@
...
+set -x
 
-# Needed to run add-apt-repository
-apt update && apt install -y software-properties-common
+export DEBIAN_FRONTEND=noninteractive
 
-add-apt-repository -y ppa:graphics-drivers
+mkdir -p /usr/lib/nvidia/
+apt update
+apt install -y --no-install-recommends libcuda1-390
```
Comment (marcoabreu) [inline tests/ci_build/install/ubuntu_install_nvidia.sh]: This package also installs nvidia-390 according to
```
[91m+ apt install -y --no-install-recommends libcuda1-390
The following additional packages will be installed:
  [...] nvidia-390
```

This means that we are back to the old state. Additionally, this does not clearly specify the cuda version.

**Comment by larroy** [comment]
@marcoabreu merge?

**Comment by larroy** [comment]
Thanks

**Pull request merged**